os/net/bluetooth/bt_util.h

```diff
@@ -0,0 +1,118 @@
+/****************************************************************************
+ * include/tinyara/bluetooth/bt_null.h
```

Comment:
Please correct the the path as well as file name.

Reply:
Corrected the file name